### PR TITLE
DI-368 Upgrade Serverless to V3

### PR DESCRIPTION
**Pull request by JackPlowman**
## Link to JIRA Ticket

- https://nhsd-jira.digital.nhs.uk/browse/DI-368

## Description

This changes the serverless version to V3, once a new docker image has been built with the correct version then an upgrade should work smoothly. 

## Type of change

- Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Testing

Please tick the testing that has been completed

- [x] Integration tests
image

### CODE REVIEW

**Review by mabe13 — APPROVED** [review]

File: build/docker/serverless/Dockerfile [hunk 1]
```diff
@@ -3,7 +3,7 @@ WORKDIR /
 # Install serverless
 RUN apk add python3 make gcc g++
 RUN export PATH="$PATH:$(yarn global bin)"
-RUN yarn global add serverless@2.72.2
+RUN yarn global add serverless
```
Comment (rohith1122) [inline build/docker/serverless/Dockerfile:6]: Are we not using a particular version here?

**Review by rohith1122 — APPROVED** [review]
left a comment about the version otherwise LGTM

**Review by rohith1122 — DISMISSED** [review]
left a comment about the version otherwise LGTM

**Review by rohith1122 — APPROVED** [review]
LGTM

**Pull request merged**